stats/lib/platform-stats-fetchers.js

```diff
@@ -67,14 +69,33 @@ export const fetchParticipantsWithTopMeasurements = async (pgPool, filter) => {
  * @param {import('./typings.js').DateRangeFilter} filter
  */
 export const fetchDailyRewardTransfers = async (pgPool, filter) => {
+  assert(
+    new Date(filter.to).getTime() - new Date(filter.from).getTime() <= 31 * ONE_DAY,
```

Comment:
Is the code calling `fetchDailyRewardTransfers` validating that filter from/to values are valid dates?

Fortunately, invalid date strings are parsed as `NaN`, and the condition here will be false, so we are covered 👍🏻 

```
> new Date('foo').getTime() - new Date('bar').getTime() <= 310000
false
```